### PR TITLE
DOMA-3197 moved delete deleted meter meter readings to worker

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**

### CODE REVIEW

File: apps/condo/domains/meter/tasks/deleteDeletedMeterMeterReadings.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const { getSchemaCtx, find } = require('@condo/keystone/schema')
+const { createTask } = require('@condo/keystone/tasks')
+const { MeterReading } = require('../utils/serverSchema')
+
+/**
+ * Soft delete meter readings after soft delete meter
+ */
+async function deleteDeletedMeterMeterReadings (deletedMeter, deletedMeterAt) {
```
Comment (AntonAL) [inline apps/condo/domains/meter/tasks/deleteDeletedMeterMeterReadings.js:8]: Could you please change name to more humanized, like 'deleteReadingsOfDeletedMeter'?

File: apps/condo/domains/meter/tasks/deleteDeletedMeterMeterReadings.spec.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const {
+    MeterReadingSource,
+    createTestMeterReading, MeterResource, createTestMeter, updateTestMeter,
+} = require('@condo/domains/meter/utils/testSchema')
+const { CALL_METER_READING_SOURCE_ID, COLD_WATER_METER_RESOURCE_ID } = require('@condo/domains/meter/constants/constants')
+
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+
+describe('Submit meter readings push notification', () => {
+    it('readings are deleted after the related meter is deleted', async () => {
```
Comment (AntonAL) [inline apps/condo/domains/meter/tasks/deleteDeletedMeterMeterReadings.spec.js:14]: Why this test case is inside "…push notifications" describe section?

File: apps/condo/domains/meter/tasks/deleteReadingsOfDeletedMeter.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+async function deleteReadingsOfDeletedMeter (deletedMeter, deletedMeterAt) {
+    const { keystone: context } = await getSchemaCtx('Property')
+
+    const meterId = deletedMeter.id
+    const meterReadings = await find('MeterReading', {
+        meter: { id: meterId },
+        deletedAt: null,
+    })
+
+    for (const reading of meterReadings) {
```
Comment (sitozzz) [inline apps/condo/domains/meter/tasks/deleteReadingsOfDeletedMeter.js:17]:
```suggestion
    for (const meterReading of meterReadings) {
```

File: apps/condo/domains/meter/tasks/deleteReadingsOfDeletedMeter.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+const { createTask } = require('@condo/keystone/tasks')
+const { MeterReading } = require('../utils/serverSchema')
+
+/**
+ * Soft delete meter readings after soft delete meter
+ */
+async function deleteReadingsOfDeletedMeter (deletedMeter, deletedMeterAt) {
+    const { keystone: context } = await getSchemaCtx('Property')
+
+    const meterId = deletedMeter.id
```
Comment (sitozzz) [inline apps/condo/domains/meter/tasks/deleteReadingsOfDeletedMeter.js:11]: looks like this variable could be safety removed

**Review by sitozzz — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**